src/front-end/typescript/lib/pages/proposal/sprint-with-us/lib/components/phase.tsx

```diff
@@ -293,12 +293,8 @@ export type Errors = CreateSWUProposalPhaseValidationErrors;
 
 /**
  * No need to set errors as the fields themselves can't result in errors.
- * TODO - find a better solution than disabling eslint
  */
-export function setErrors(
-  state: Immutable<State>,
-  errors?: Errors // eslint-disable-line @typescript-eslint/no-unused-vars
-): Immutable<State> {
+export function setErrors(state: Immutable<State>): Immutable<State> {
   return state;
```

Comment:
generates compile errors in `src/front-end/typescript/lib/pages/proposal/sprint-with-us/lib/components/team.tsx` lines 198,199, 201 - 'Expected 1 arguments, but got 2'`

Reply:
seems strange though, I wonder where we're passing it. We can probably remove it in that call